lndc/netio.go

```diff
@@ -1,93 +0,0 @@
-package lndc
-
-import (
-	"bytes"
-	"encoding/binary"
-	"fmt"
-	"io"
-	"net"
-)
-
-// maxMsgSize is the longest message supported.  This is payload size, not
-// encrypted size.
-const maxMsgSize = 1 << 24
```

Comment:
We need to keep this; DLC message are using it.
(though I suppose it could be (1<<24)-1 instead)